src/include/settings/settings_defs.h

```diff
@@ -66,17 +66,6 @@ SETTING_int(
     terrier::settings::Callbacks::NoOp
 )
 
-// Number of worker pool threads
```

Comment:
Did this setting go somewhere?

Reply:
It doesn't really have any meaning until we architect the task/thread infrastructure so I removed it.